src/utils.js

```diff
@@ -154,9 +155,9 @@ export function toWidgetEditable( editable, writer ) {
 
 	editable.on( 'change:isFocused', ( evt, property, is ) => {
 		if ( is ) {
-			writer.addClass( 'ck-editable_focused', editable );
+			writer.addClass( 'ck-editor__nested-editable_focused', editable );
```

Comment:
At first I thought – why using the more specific class here? But then I realised that the main editable gets `.ck-focused` so it's a different case anyway.